sdk/storage/azure-storage-blob/src/main/java/com/azure/android/storage/blob/models/ContainerDeleteHeaders.java

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.storage.blob.models;
+
+import com.azure.android.core.util.DateTimeRfc1123;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
+
+import org.threeten.bp.OffsetDateTime;
+
+/**
+ * Defines headers for Delete operation.
+ */
+@JacksonXmlRootElement(localName = "Container-Delete-Headers")
+public final class ContainerDeleteHeaders {
+    /*
```

Comment:
I mentioned this in other PRs: I think we can add Javadoc entries at the class members level to avoid repeating this in the headers and setters, and referencing to the value itself. Would it not play nice with private values when generating the docs?
```suggestion
    /**
```